viz_scripts/generic_metrics.ipynb

```diff
@@ -147,15 +147,19 @@
     "file_name= 'ntrips_commute_mode_confirm%s' % file_suffix\n",
     "\n",
     "try:\n",
-    "    labels_mc = expanded_ct.query(\"Trip_purpose == 'Work'\").Mode_confirm.value_counts(dropna=True).keys().tolist()\n",
-    "    values_mc = expanded_ct.query(\"Trip_purpose == 'Work'\").Mode_confirm.value_counts(dropna=True).tolist()\n",
-    "    commute_quality_text = scaffolding.get_quality_text(expanded_ct, expanded_ct.query(\"Trip_purpose == 'Work'\"), \"commute\", include_test_users)\n",
+    "    if (len(dynamic_labels)):\n",
+    "        trip_purpose_query = \"Trip_purpose == 'To Work'\"\n",
```

Comment:
Agreed, the mapping has been updated to use scaffolding.mapping_labels(dynamic_labels, PURPOSE) instead to identify the appropriate translated string instead of just calling 'To Work'.